CHANGELOG.md

```diff
@@ -48,6 +48,12 @@ New modules
 Additions to existing modules
 -----------------------------
 
+* In `Data.List.Properties`:
+  ```agda
+  nonZero-product : All NonZero ns → NonZero (product ns)
```

Comment:
I think that `product≢0 ` is a better name, for consistency with other `*≢0 ` lemmas in `Data.Nat.Properties`?

Reply:
And that also means that [`Data.Nat.Primality`](https://github.com/agda/agda-stdlib/blob/0da5ba7629ca1a911958f155fbecf2cc85646c4a/src/Data/Nat/Primality.agda#L325-L330) can be trivially refactored to make use of this lemma...